src/app/projects/states/dashboard/directives/task-dashboard/directives/task-due-card/task-due-card.component.html

```diff
@@ -0,0 +1,89 @@
+
+<div *ngIf="(!task.inFinalState() || (task.inTimeExceeded() && !task.isPastDeadline())) && !task.inAwaitingFeedbackState()"> <!--Only show this if it matters... not in final or waiting for tutor input-->
+  <div *ngIf="task.isDueSoon() && !task.pastDueDate()"> <!--Due soon-->
+    <div class="card card-warning card-sm">
+      <div class="card-heading">
+        <h4>Aim To Complete Soon - Due in {{task.timeUntilDueDateDescription()}}</h4>
+      </div>
+      <div class="card-body" [hidden]="task.inDiscussState()"> <!--Due and not in discuss-->
+        <p>
+          This task's due date is <strong>{{task.localDueDateString()}}</strong>.
+          You should aim to complete this task before then to keep your progress on track.
+        </p>
+      </div>
+      <div class="card-body" [hidden]="!task.inDiscussState()"> <!--Due but in discuss-->
+        <p>
+          This task's due date is <strong>{{task.localDueDateString()}}</strong>.
+          Make sure to discuss this task with your tutor as soon as possible.
+        </p>
+        <p class="text-muted">
```

Comment:
These old bootstrap css classes need to be refactored using Angular Material Typography